modules/remove.js

```diff
@@ -0,0 +1,35 @@
+import { setDataLocalStorage, getDataLocalStorage } from './store.js';
+import { refreshIndex, renderTaskDom } from './render.js';
+import { refreshDragDropTarget } from './drag-drop.js';
+import { refreshDescriptions } from './description.js';
+
+export const removeTask = (event) => {
+  if (event.target.classList.contains('remove')) {
```

Comment:
I guess that if this line was separated in two different lines, it would be better for understanding:

`const containsRemove = event.target.classList.contains('remove');
  if (containsRemove) {
 `